resources/xip-patch/templates/clusterrole.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: xip-patch
```

Comment:
The name element in the chart should contain the Release.name too: `{{ .Release.Name }}-xip-patch`